apps/meteor/app/file-upload/server/methods/sendFileMessage.ts

```diff
@@ -64,8 +64,8 @@ export const parseFileIntoMessageAttachments = async (
 			attachment.image_preview = await FileUpload.resizeImagePreview(file);
 			const thumbResult = await FileUpload.createImageThumbnail(file);
 			if (thumbResult) {
-				const { data: thumbBuffer, width, height } = thumbResult;
-				const thumbnail = await FileUpload.uploadImageThumbnail(file, thumbBuffer, roomId, user._id);
+				const { data: thumbBuffer, width, height, thumbFile } = thumbResult;
+				const thumbnail = await FileUpload.uploadImageThumbnail(thumbFile, thumbBuffer, roomId, user._id);
```

Comment:
And finally, following my previous comments, here we can change the first parameter to instead of providing an `IUpload` we can provide an object like: `{ thumbFileName, thumbFileType, originalFileId }`.

It will improve the readability since it does not change the file directly inside the functions.

Reply:
Hey @MarcosSpessatto I have added the required changes.